node-runner-cli/config/EnvVars.py

```diff
@@ -20,6 +20,9 @@
 SUPPRESS_API_COMMAND_WARN = "SUPPRESS_API_COMMAND_WARN"
 PROMPT_FEEDS = "PROMPT_FEEDS"
 CORE_DOCKER_REPO_OVERRIDE = "CORE_DOCKER_REPO_OVERRIDE"
+GATEWAY_DOCKER_REPO_OVERRIDE = "GATEWAY_DOCKER_REPO_OVERRIDE"
```

Comment:
Since we are refactoring, can we put all env vars in the config files directly? sounds like duplicating the work to have them in an EnvVars.py. Not to mention that they are all mixed up :)

Reply:
where do you think they are duplicated? These are env vars are there to change the behaviour of the node cli. When you refer config, I take you are taking about tht config file that we create from the CLI.